content/aws_marketplace.md

```diff
@@ -42,19 +43,38 @@ AWS provides 5 VPCs for each region. If you require more VPCs, please contact [A
 
 1. Enter the following values for your deployment.
 
-     - Stack Name: `Chef-Automate`
-     - EC2RootVolumeSize: `Default: 40`
-     - Instance Type:`Default: t2.xlarge`
-     - KeyName: _Enter your existing keypair_
-     - SecurityGroupCidrIp: `0.0.0.0/0`
-     - SubnetCIDR: `10.0.0.0/24`
-     - VpcCIDR: `10.0.0.0/16`
-
-1. Select **Next** and create your Chef Automate deployment. This process can take several minutes.
+    ![Enter required information and click next](/images/stack_details.png "Stack Details")
 
 1. Give Chef Automate an additional five minutes for all the services to start running.
 
-![Select next to create stack](/images/StackDetails.png "Stack Details")
+1. Configure stack options:
+
+    - **Tags:** Create a tag that can be used to refer to your resources (for example, `Key: Name, Value: Chef-automate-stack`).
```

Comment:
With `Key: Name, Value: Chef-automate-stack`, reserve the inline code for text that the user is going to enter into the Web UI. So: 

"Key: `Name`, Value: `Chef-automate-stack`."